app/views/filters/study_type/index.html.erb

```diff
@@ -0,0 +1,78 @@
+<%= content_for :page_title, "Filter by study type" %>
+<% content_for :before_content do %>
+  <a href="<%= results_path %>" class="govuk-back-link">Back to search results</a>
```

Comment:
We need to pass the params back here. There is a helper for this too in `app/helpers/view_helper.rb`. We need to link to the referrer I think (and protect against an [open redirect](https://owasp.org/www-project-cheat-sheets/cheatsheets/Unvalidated_Redirects_and_Forwards_Cheat_Sheet.html)) I can do it in mine and update here if we can hold off deploying until I'm done.